Errors in `make test` command

In case there are no test files in package folders `make test` command fails because `xargs -l` passing nothing to sh.

Steps to reproduce the behavior:
1. Run `make test`
2. See the errors:
```sh
sh: -c: option requires an argument
make: *** [test] Error 123
```

**Expected behaviour**
No errors in output

**Desktop (please complete the following information):**
 - OS: Mac OS Mojave 10.14.2
 - Go version: 1.11.4

--- a/pkg/helper/helper_test.go
+++ b/pkg/helper/helper_test.go
@@ -1,0 +1,33 @@
+package helper
+
+import (
+	"bytes"
+	"errors"
+	"log"
+	"testing"
+)
+
+func TestLogE(t *testing.T) {
+	var data bytes.Buffer
+
+	log.SetOutput(&data)
+	log.SetFlags(0)
+	LogE("new message", nil)
+	expected := ""
+	if data.String() != expected {
+		t.Errorf("Expected message %s, got %s", expected, data.String())
+	}
+	LogE("message", errors.New("Error"))
+	expected = "message: Error\n"
+	if data.String() != expected {
+		t.Errorf("Expected message %s, got %s", expected, data.String())
+	}
+	LogE("new message", nil)
+}
+
+func TestLogF(t *testing.T) {
+	LogF("message", nil)
+	if t.Failed() {
+		t.Error("Expected success, got failed")
+	}
+}

